Gemfile-rails-dependencies

```diff
@@ -23,17 +23,31 @@ when /stable$/
   gem 'activerecord-jdbcsqlite3-adapter', git: 'https://github.com/jruby/activerecord-jdbc-adapter', platforms: [:jruby]
 
   gem_list.each do |rails_gem|
-    gem rails_gem, :git => "https://github.com/rails/rails.git", :branch => version
+    if version == '5-2-stable' && RUBY_VERSION < '2.3'
+      # Until 5-2-stable is fixed on Ruby 2.2, pin to last working version
```

Comment:
I thought it was broken on `5.2.x`, and was fine on `5-2-stable`. As far as I recall, it was later fixed on `5.2.x`, too (https://github.com/rspec/rspec-rails/commit/2b8fffed490125c5b08de36177765cb0cb234037#). Did it break again, now on `5-2-stable`?